sw/device/silicon_creator/mask_rom/mask_rom_start.S

```diff
@@ -211,6 +211,9 @@ _mask_rom_start_boot:
   sw t0, CSRNG_CTRL_REG_OFFSET(a0)
 
   li a0, TOP_EARLGREY_EDN0_BASE_ADDR
+  // Set the generate length to maximum for the boot generate command.
+  li t0, 0x7ffff003
```

Comment:
@moidx I believe this is what you mentioned offline, right? For reference, the reset value of this register is `0xfff003`.

According to the csrng spec, MSb is reserved, glen (generate length) is 19 bits long, and its max value is 2^19. But I think this is a typo since the max value can be at most 2^19 -1 (glen is 19 bits long), which is what I used here.

I wish we had macros for this as well. Wanted to confirm before creating an enhancement issue.

Reply:
Yes, this is the register I was referring to. I agree we should create some macros to generate CSRNG commands.

Reply:
Thanks! Created #12453 for macros.

Reply:
and #12454 for the typo in the spec.

Reply:
fyi: dropped this commit as we discussed online after the doc fix in #12466. `glen` in reset value of `BOOT_GEN_CMD` is only one less than the max. I think it's safer to just use that instead of hardcoding a magic constant here.